doc/src/site/sphinx/tutorials/secured_flow.rst

```diff
@@ -113,31 +143,54 @@ certificates are created.
         .. code:: scala
 
             import org.apache.spark.h2o._
-            val conf = new H2OConf().setAutoFlowSslEnabled()
+            val conf = new H2OConf().setAutoFlowSslEnabled().setVerifySslCertificates(false)
             val hc = H2OContext.getOrCreate(conf)
 
 
     .. tab-container:: Python
         :title: Python
 
-        To enable https in Pysparkling using this mode, you can start PySparkling as:
+        To enable https in PySparkling using this mode, you can start PySparkling as:
 
         .. code:: shell
 
-            bin/pysparkling --conf "spark.ext.h2o.auto.flow.ssl=true"
+            bin/pysparkling --conf "spark.ext.h2o.auto.flow.ssl=true"  --conf "spark.ext.h2o.verify_ssl_certificates=false"
 
         and when you have the shell running, start ``H2OContext`` as:
 
         .. code:: python
 
             from pysparkling import *
-            hc = H2OContext.getOrCreate(verify_ssl_certificates=False)
+            hc = H2OContext.getOrCreate()
 
         You can also start PySparkling shell without the configuration
         and specify it using the setters on ``H2OConf`` as:
 
         .. code:: python
 
             from pysparkling import *
-            conf = H2OConf().setAutoFlowSslEnabled()
-            hc = H2OContext.getOrCreate(conf, verify_ssl_certificates=False)
+            conf = H2OConf().setAutoFlowSslEnabled().setVerifySslCertificates(False)
+            hc = H2OContext.getOrCreate(conf)
+
+    .. tab-container:: R
+        :title: R
+
+        To enable https in RSparkling using this mode, run in your RStudio:
+
+
+        and when you have the shell running, start ``H2OContext`` as:
+
+        .. code:: r
+
+            from pysparkling import *
```

Comment:
`pysparkling` seems to be a wrong namespace for R.

Reply:
Yup, thanks, rebase error